include/libopencm3/stm32/f7/pwr.h

```diff
@@ -0,0 +1,284 @@
+
+/* Bits [7:4]: Reserved, must be kept at reset value. */
+
+/* BRR: Backup regulator ready */
+#define PWR_CSR1_BRR			(1 << 3)
+
+/* PVDO: PVD output */
+#define PWR_CSR1_PVDO			(1 << 2)
+
+/* SBF: Standby flag */
+#define PWR_CSR1_SBF			(1 << 1)
+
+/* WUIF: Wakeup internal flag */
+#define PWR_CSR1_WUIF			(1 << 0)
+
+/* --- PWR_CR2 values ------------------------------------------------------ */
+
+/* Bits [31:14]: Reserved, must be kept at reset value. */
+
+/* WUPP6: Wakeup pin polarity bit for PI11 */
+#define PWR_CR2_WUPP6			(1 << 13)
+
+/* WUPP5: Wakeup pin polarity bit for PI8 */
+#define PWR_CR2_WUPP5			(1 << 12)
+
+/* WUPP4: Wakeup pin polarity bit for PC13 */
+#define PWR_CR2_WUPP4			(1 << 11)
+
+/* WUPP3: Wakeup pin polarity bit for PC1 */
+#define PWR_CR2_WUPP3			(1 << 10)
+
+/* WUPP2: Wakeup pin polarity bit for PA2 */
+#define PWR_CR2_WUPP2			(1 << 9)
+
+/* WUPP1: Wakeup pin polarity bit for PA0 */
+#define PWR_CR2_WUPP1			(1 << 8)
+
+/* Bits [7:6]: Reserved, must be kept at reset value. */
+
+/* CWUPF6: Clear Wakeup Pin flag for PI11 */
+#define PWR_CR2_CWUPF6			(1 << 5)
+
+/* CWUPF5: Clear Wakeup Pin flag for PI8 */
+#define PWR_CR2_CWUPF5			(1 << 4)
+
+/* CWUPF4: Clear Wakeup Pin flag for PC13 */
+#define PWR_CR2_CWUPF4			(1 << 3)
+
+/* CWUPF3: Clear Wakeup Pin flag for PC1 */
+#define PWR_CR2_CWUPF3			(1 << 2)
+
+/* CWUPF2: Clear Wakeup Pin flag for PA2 */
+#define PWR_CR2_CWUPF2			(1 << 1)
+
+/* CWUPF1: Clear Wakeup Pin flag for PA0 */
+#define PWR_CR2_CWUPF1			(1 << 0)
+
+/* --- PWR_CSR2 values ------------------------------------------------------ */
+
+/* Bits [31:14]: Reserved, must be kept at reset value. */
+
+/* EWUP6: Enable Wakeup pin for PI11 */
+#define PWR_CSR2_EWUP6			(1 << 13)
+
+/* EWUP5: Enable Wakeup pin for PI8 */
+#define PWR_CSR2_EWUP5			(1 << 12)
+
+/* EWUP4: Enable Wakeup pin for PC13 */
+#define PWR_CSR2_EWUP4			(1 << 11)
+
+/* EWUP3: Enable Wakeup pin for PC1 */
+#define PWR_CSR2_EWUP3			(1 << 10)
+
+/* EWUP2: Enable Wakeup pin for PA2 */
+#define PWR_CSR2_EWUP2			(1 << 19)
+
+/* EWUP1: Enable Wakeup pin for PA0 */
+#define PWR_CSR2_EWUP1			(1 << 18)
+
+/* Bits [7:6]: Reserved, must be kept at reset value. */
+
+/* WUPF6: Wakeup Pin flag for PI11 */
+#define PWR_CSR2_WUPF6			(1 << 5)
+
+/* WUPF5: Wakeup Pin flag for PI8 */
+#define PWR_CSR2_WUPF5			(1 << 4)
+
+/* WUPF4: Wakeup Pin flag for PC13 */
+#define PWR_CSR2_WUPF4			(1 << 3)
+
+/* WUPF3: Wakeup Pin flag for PC1 */
+#define PWR_CSR2_WUPF3			(1 << 2)
+
+/* WUPF2: Wakeup Pin flag for PA2 */
+#define PWR_CSR2_WUPF2			(1 << 1)
+
+/* WUPF1: Wakeup Pin flag for PA0 */
+#define PWR_CSR2_WUPF1			(1 << 0)
+
+/* --- Function prototypes ------------------------------------------------- */
+
+enum pwr_vos_scale {
+	PWR_SCALE1, // <= 180MHz w/o overdrive, <= 216MHz w/ overdrive
+	PWR_SCALE2, // <= 168MHz w/o overdrive, <= 180MHz w/ overdrive
+	PWR_SCALE3, // <= 144MHz
+};
+
+BEGIN_DECLS
+
+void pwr_set_vos_scale(enum pwr_vos_scale scale);
```

Comment:
Is this the same function name and enum name used in the other families?  (L0/L1/L4 to memory all have range scales)